lib/scope.js

```diff
@@ -1035,6 +1219,8 @@ module.exports = {
       await scope.afterStep(scope, { waitForShowIf: true });
     },
     CANVAS: async function ( scope, { handle, set_to }) {
+      // `<canvas>` takes a while to load
+      await scope.page.waitForSelector('#dasigcanvas');
```

Comment:
Signature stuff.

Reply:
Waits here too because of the possibility that this may be called from elsewhere.